src/main/kotlin/dynamic_programming/PalindromePartitioning.kt

```diff
@@ -0,0 +1,75 @@
+package dynamic_programming
+
+/**
+ * Palindrome Partitioning Algorithm
+ *
+ * You are given a string as input, and task is to find the minimum number of partitions to be made,
+ * in the string sot that the resulting strings are all palindrome
+ *
+ * Time Complexity = O(n^2)
+ *
```

Comment:
Add about author 
@author [name](github url)

Reply:
according to contribution guidelines, we are supposed to do that, our commit is sufficient to mark our contribution